mne/io/ctf/tests/test_ctf.py

```diff
@@ -216,6 +218,42 @@ def test_read_ctf():
                   op.join(ctf_dir, ctf_fname_continuous), 'foo')
 
 
+@testing.requires_testing_data
+def test_rawctf_clean_names():
+    """Test RawCTF _clean_names method"""
+    # read test data
+    raw = read_raw_ctf(op.join(ctf_dir, ctf_fname_catch))
+    raw_cleaned = read_raw_ctf(op.join(ctf_dir, ctf_fname_catch),
+                               clean_names=True)
+    test_channel_names = _clean_names(raw.ch_names)
+    test_info_comps = copy.deepcopy(raw.info['comps'])
+
+    # channel names should not be cleaned by default
+    assert not array_equal(raw.ch_names, test_channel_names)
```

Comment:
these are just `list of str` correct? If so please use `assert raw.ch_names != test_channel_names`. It gives a more informative error if it fails.